### PR TITLE
fix: map properties error

### PR DESCRIPTION
**Pull request by YongwuHe**

### CODE REVIEW

File: arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/util/StringUtil.java [hunk 1]
```diff
@@ -106,6 +106,17 @@ private static void indent(StringBuilder sb, int indent) {
         }
     }
 
+    public static String mapToString(Map<String, String> map) {
```
Comment (mr3) [inline arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/util/StringUtil.java:109]: add `MapUtils.isEmpty` return `StringUtil#EMPTY`

File: arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/model/ArexMocker.java [hunk 2]
```diff
@@ -35,11 +35,11 @@ public class ArexMocker implements Mocker {
     public ArexMocker() {
     }
 
-    public ArexMocker(MockCategoryType categoryType) {
+    public ArexMocker(MockCategoryType categoryType, Map<String, String> arexMockerTags) {
```
Comment (mr3) [inline arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/model/ArexMocker.java:38]: rename to `mockTags`

**Review by lucas-myx — DISMISSED** [review]

File: arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/model/ArexMocker.java (hunk 2 above)
Comment (mr3) [inline arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/model/ArexMocker.java:38]: rename to `mockTags`

File: arex-instrumentation-api/src/main/java/io/arex/inst/runtime/config/Config.java [hunk 3]
```diff
@@ -47,6 +47,7 @@ public static Config get() {
     private final String recordVersion;
     private final Set<String> includeServiceOperations;
     private final Set<String> coveragePackages = new ConcurrentHashSet<>();
+    private final Map<String, String> arexMockerTags;
```
Comment (mr3) [inline arex-instrumentation-api/src/main/java/io/arex/inst/runtime/config/Config.java:50]: rename to `mockTags`

**Review by mr3 — APPROVED** [review]

**Review by lucas-myx — APPROVED** [review]

**Pull request merged**